style.css

```
@@ -0,0 +1,131 @@
+:root {
+  --font-family-anton: "Anton", sans-serif;
+  --font-family-noto-sans-jp: "Noto Sans JP", sans-serif;
+
+  --color-white: #fffefc;
+  --color-black: #020911;
+  --color-accent-red: #f20574;
+  --color-accent-yellow: #f2e52e;
+  --color-background-black: #171525;
+  --color-background-navy: #030427;
+  --color-form-background-yellow: #fffab9;
+  --color-form-placeholder-gray: #ad9783;
+  --color-form-border-red: #df6da2;
+  --color-button-hover-red: #f043a5;
+  --color-course-hover-yellow: #f4e877;
+  --color-hero-overlay: rgba(57, 24, 111, 0.4);
+  --color-course-overlay: rgba(32, 8, 19, 0.63);
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: var(--font-family-noto-sans-jp);
+}
+
+img {
+  display: block;
+}
+
+/* header */
+.header {
+  background: var(--color-background-black);
+  height: 80px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 10;
+  width: 100%;
+}
+
+.header-logo-wrapper {
+  margin: 0;
+}
+
+.header-logo {
+  height: 50px;
+}
+
+@media (max-width: 750px) {
+  .header {
+    height: 60px;
+  }
+  .header-logo {
+    height: 30px;
+  }
+}
+
+/* hero */
+.hero {
+  background-image: url(./img/hero.png);
+  background-size: cover;
+  background-position: 10px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 94px 40px;
```

```suggestion
  padding: 96px 40px;
```
Figmaだと、上下のpaddingは「96px」ですね

---

修正しました。